modules/standard/Random.enc

```diff
@@ -37,4 +37,9 @@ read class Random
 
     END
   end
```

Comment:
I fixed it all up by hand. 
(Didn't the return key work when writing the pretty printer? :smiley:)